datalad/local/tests/test_copy_file.py

```diff
@@ -339,4 +339,4 @@ def test_copy_file_prevent_dotgit_placement(srcpath, destpath):
             [sub.pathobj / '.git' / 'config',
              dest.pathobj / 'some', '.git'], on_failure='ignore'),
         status='impossible',
-        action='copy')
+        action='copy_file')
```

Comment:
Just a heads up: when I tried to test this simple update locally, this test errors for me with

```
======================================================================
ERROR: datalad.local.tests.test_copy_file.test_copy_file_prevent_dotgit_placement
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/kyle/src/python/venvs/datalad/lib/python3.7/site-packages/nose/case.py", line 198, in runTest
    self.test(*self.arg)
  File "/home/kyle/src/python/datalad/datalad/tests/utils.py", line 691, in newfunc
    return t(*(arg + (filename,)), **kw)
  File "/home/kyle/src/python/datalad/datalad/tests/utils.py", line 691, in newfunc
    return t(*(arg + (filename,)), **kw)
  File "/home/kyle/src/python/datalad/datalad/local/tests/test_copy_file.py", line 340, in test_copy_file_prevent_dotgit_placement
    dest.pathobj / 'some', '.git'], on_failure='ignore'),
  File "/home/kyle/src/python/datalad/datalad/distribution/dataset.py", line 499, in apply_func
    return f(**kwargs)
  File "/home/kyle/src/python/datalad/datalad/interface/utils.py", line 494, in eval_func
    return return_func(generator_func)(*args, **kwargs)
  File "/home/kyle/src/python/datalad/datalad/interface/utils.py", line 482, in return_func
    results = list(results)
  File "/home/kyle/src/python/datalad/datalad/interface/utils.py", line 413, in generator_func
    allkwargs):
  File "/home/kyle/src/python/datalad/datalad/interface/utils.py", line 552, in _process_results
    for res in results:
  File "/home/kyle/src/python/datalad/datalad/local/copy_file.py", line 286, in __call__
    for res in _copy_file(src_file, dest_file, cache=repo_cache):
  File "/home/kyle/src/python/datalad/datalad/local/copy_file.py", line 470, in _copy_file
    eval_file_type=True,
KeyError: 'dictionary is empty'
```

My first thought is that it's due to the (newer) git-annex version that I have locally, but I haven't yet looked into it.